bin/configurator/static/_includes/en/configurator/partials/ci/gitlab_docker_main_section.md.liquid

```diff
@@ -18,17 +18,44 @@ Follow [official instructions](https://docs.gitlab.com/runner/register/index.htm
 
```

Comment:
You sure Installing and Registering Gitlab Runner sections should be left as is?